modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml

```diff
@@ -3624,6 +3627,177 @@ paths:
                       - HeadlessCommerceAdminCatalog.write
             summary: Creates or updates a option category.
             tags: ["OptionCategory"]
+    "/optionCategories/by-externalReferenceCode/{externalReferenceCode}":
+        delete:
+            operationId: deleteOptionCategoryByExternalReferenceCode
+            parameters:
+                - in: path
+                  explode: false
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      description:
+                          The resource external reference code
+                      example: AB-34098-789-N
+                      minLength: 1
+                      type: string
+                  style: simple
+            responses:
+                204:
+                    description:
+                        No content
```

Comment:
is this the new way? I remember something like returning a {} ... or is it the opposite?